### DISCUSSION TITLE
Pouvoir modifier ma fiche produit depuis l'espace membre

### DISCUSSION THREAD

**Discussion by astranchet** [betagouv/cap#7] [Besoin]
Aujourd'hui, certains champs ne sont pas modifiables depuis l'espace membre (par exemple le champ "équipe"). Ça serait bien que TOUT soit modifiable pour ne plus rien gérer dans github

**Comment by revolunet** [comment]
a priori cette PR permet de résoudre ce sujet https://github.com/betagouv/espace-membre-next/pull/189

**Reply by sneko** [reply]
@astranchet est-ce que ça répond à ton besoin ?
